src/components/BusinessController/index.js

```diff
@@ -90,6 +92,12 @@ export const BusinessController = (props) => {
     const hour = timeToCloseFormatted?.split(':')[0]
     const minute = timeToCloseFormatted?.split(':')[1]
     const result = currentHour > hour || (currentHour === hour && currentMinute >= minute)
+    const timeDifference = (new Date(null, null, null, hour, minute) - new Date(null, null, null, currentHour, currentMinute)) / 60000
+    if (timeDifference <= 30 && timeDifference > 0) {
+      setBusinessWillCloseSoonMinutes(timeDifference)
+    } else {
+      setBusinessWillCloseSoonMinutes(null)
+    }
```

Comment:
You can avoid if sentences with:

`setBusinessWillCloseSoonMinutes(timeDifference <= 30 && timeDifference > 0 ? timeDifference  : null)`